src/components/explore/components/SearchResults.tsx

```diff
@@ -52,10 +57,18 @@ function SearchResults({ results }: { results: SearchComicContent[] }) {
             <img
               src={`https://meo3.comick.pictures/${searchResult?.md_covers?.[0]?.b2key}`}
               alt={`Comic Cover Image`}
-              className="rounded-lg h-full w-full shadow-custom"
+              className="rounded-lg h-4/5 w-1/4 object-fit ml-2 md:w-1/5 lg:w-1/10"
             ></img>
-            <div className="image-overlay h-full w-full rounded-lg bg-gradient-to-t from-plum absolute flex items-end justify-center text-center p-0 m-0">
-              <p className="p-2 font-sans text-white">{searchResult?.title}</p>
+
+            <div className="comic-quick-info-container h-full px-2 py-1 w-4/5">
+              <p className="p-2 w-full text-wrap font-sans text-white font-semibold">
+                {searchResult?.title}
+              </p>
+              <p className="p-2 w-full text-wrap font-sans text-white">
+                {searchResult.desc
+                  ? `${searchResult?.desc.substring(0, 50)}...`
+                  : null}
```

Comment:
just make sure there is no awkward styling with the div being empty when it returns `null`